WFP-VAM/prism-app
# Add location name to chart title

### Description

This fixes #1195 .



## How to test the feature:

- Navigate to charts panel
- Notice country name is in the title if no admin is selected, else the admin's name is displayed 

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:

<img width="995" alt="Screenshot 2024-04-20 at 10 13 48 AM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/236db651-5288-4245-9537-9d104e6f95cf">
<img width="981" alt="Screenshot 2024-04-20 at 10 14 14 AM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/7536c6e6-95d5-43e4-9890-84ee32929d39">
<img width="898" alt="Screenshot 2024-04-20 at 10 15 00 AM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/5a0ac630-4511-4d36-adff-7edbd7394fff">
